c2corg_ui/static/js/maincontroller.js

```diff
@@ -76,3 +91,17 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 }
+
+
+/**
+ * @export
+ */
+app.MainController.prototype.redirect = function(url) {
```

Comment:
The function name is not explicit enough if outside of the ``editButton`` controller.